src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentState.cs

```diff
@@ -28,34 +28,51 @@ internal partial class DocumentState
     private Task<TextAndVersion>? _loaderTask;
     private SourceText? _sourceText;
     private VersionStamp? _version;
+    private readonly int _numericVersion;
```

Comment:
Consider using `VersionStamp` rather than a number and rename `_version` to `_textVersion`.